website/static/js/share/sideBar.js

```diff
@@ -0,0 +1,91 @@
+var m = require('mithril');
+var $osf = require('js/osfHelpers');
+var utils = require('./utils.js');
+
+var SideBar = {};
+
+SideBar.view = function(ctrl){
+    if (ctrl.vm.results === null){
+        return [];
+    }
+    return m('', [
```

Comment:
That is what is created on the page. I'll see if I can find the documentation for it though.